pages/ksphere/kommander/1.2/install-airgapped/index.md

```diff
@@ -51,8 +51,10 @@ Installing air-gapped Kommander does not require any changes in the `inventory.y
 
 The `cluster.yaml` file provides the configuration details for creating your Konvoy cluster. Installing Kommander in an air-gapped environment requires additional configuration. Ensure the `cluster.yaml` contains all the changes outlined in [air-gapped Konvoy installation][https://docs.d2iq.com/ksphere/konvoy/1.6/install/install-airgapped/#configure-the-image-registry] documentation. Follow this procedure:
 
+**TODO**: How is the reader supposed to conduct these following 3 steps?
```

Comment:
@vespian any hints?

Reply:
I believe that the line above already explains it:

`Ensure the `cluster.yaml` has all the changes outlined in [air-gapped Konvoy installation][https://docs.d2iq.com/ksphere/konvoy/1.6/install/install-airgapped/#configure-the-image-registry] documentation. Follow this procedure:`

What exactly is the difficulty here?

Reply:
The difficulty is to understand how the customer is supposed to conduct each of the steps. The reader sees a list of actions like "Ensure Kommander can use the self-hosted charts repository" but nothing tells him **how** to do that.

Reply:
>but nothing tells him how to do that.

We are describing here changes in cluster.yaml. Should we tell our customers how to edit a text file? Maybe we can simply be more clear that all these steps are done by changing the cluster.yaml?

Reply:
Ok, so all 3 checks here are satisfied by the `cluster.yaml` edit outlined below?

Reply:
Yes!